pkg/accountmanager/accountdestruction.go

```diff
@@ -1,4 +1,4 @@
-package accountwallet
+package accountmanager
```

Comment:
How about calling this the walletmanager package instead? The structure we have now is walletmanager contains wallets, and wallets contain accounts and delegations owned by the keys generated by that wallet.